util/pagination/pagination.js

```diff
@@ -0,0 +1,7 @@
+class PaginationEmbed {
```

Comment:
This class is useless as it's not constructable and it has no propery, you don't need this.